app/controllers/courses_controller.rb

```diff
@@ -122,6 +122,12 @@ def alerts
     @alerts = current_user&.admin? ? @course.alerts : @course.public_alerts
   end
 
+  def suspected_plagiarism
```

Comment:
This method isn't exercised in the specs. I think the ideal thing here would be a new feature spec that loads the activity tab and then clicks the subnav link and looks for a result in the table, since that will also touch your frontend changes. You can see an example of a similar minimal feature spec in `spec/features/article_viewer_spec.rb`.

Reply:
I've added a new feature to test this, one thing that I couldn't understand though is how to test admin / normal user? It seems it's loging as admin by default.